setup/src/main/java/io/token/setup/Application.java

```diff
@@ -11,6 +11,8 @@
 import java.nio.file.Path;
 import java.nio.file.Paths;
 
+// Comment(Colin): Can we add an alternative way to run, via a gradle task? Much nicer than having
```

Comment:
I couldn't figure out a way; then again, my gradle knowledge is puny. I tried :setup:run and then :app:run . This created ./setup/keys in setup and tried to read ./app/keys in the app. So I fell back to shadowjars.